### ISSUE TITLE
Citation is not correct

### ISSUE THREAD

**Issue by christophfriedrich** [sjmgarnier/viridis#85]
When typing `citation("viridis")`, the output is the following:

```
To cite viridis/viridisLite in publications use:

  Simon Garnier, Noam Ross, Robert Rudis, Antônio P. Camargo, Marco Sciaini, and Cédric Scherer (2021). Rvision -
  Colorblind-Friendly Color Maps for R. R package version 0.6.2.

A BibTeX entry for LaTeX users is

  @Manual{,
    title = {{viridis} - Colorblind-Friendly Color Maps for R},
    author = {{Garnier} and {Simon} and {Ross} and {Noam} and {Rudis} and {Robert} and {Camargo} and Antônio Pedro and {Sciaini} and {Marco} and {Scherer} and {Cédric}},
    year = {2021},
    note = {R package version 0.6.2},
    url = {https://sjmgarnier.github.io/viridis/},
    doi = {10.5281/zenodo.4679424},
    doi = {10.5281/zenodo.4679424},
  }
```

There's several issues here:

- [ ] Incorrect package name in plaintext citation
- [ ] Messed up names in bibtex citation
- [ ] Duplicate `doi` entry in bibtex citation

Also note that many styles don't output a `url` when a `doi` is supplied (because they turn the DOI into a link and think that one is enough), so you might want to scrap the latter in favour of the former?

**Comment by sjmgarnier** [comment]
@christophfriedrich Thanks for catching that. The "Rvision" is probably a stupid copy/paste error. The date is supposed to be generated based on the package version, but it looks like the script is not doing what it's supposed to do. I'll take a look at it as soon as I can. 

The DOI entry is not duplicated. These are actually two separate DOIs, one for viridis and one for viridisLite. 

I'm not sure to understand what you mean by "so you might want to scrap the latter in favour of the former?". Can you elaborate? Thanks.

**Comment by sjmgarnier** [comment]
Fixed with #86

**Comment by christophfriedrich** [comment]
Hi, sorry I didn't see your replies back when you were working on it. For the record, with "scrap the latter in favour of the former" I meant: When a `doi` is supplied, many styles turn that into a clickable link, and as a result of that _ignore_ a possible `url` field (because they find one clickable link to be enough). So if you would like the homepage to _always_ be rendered in citations, you would have to delete the `doi` field. Which I suggested because in my opinion the homepage is the more valuable resource, especially as it links to the Zenodo anyway. On the other hand, the DOI refers to the code of a specific version, so it _is_ the more desirable resource from a reference point of view. I think I'd change my opinion and now say that having both in the citation output is good -- the referencing person can/should still double-check what ends up in their bibliography.
